src/components/TransactionalInfo.tsx

```diff
@@ -60,7 +59,7 @@ const TransactionalInfo = ({
   const { addressHash: addressHashParam = '' } = useParams<{ addressHash: AddressHash }>()
   const addressHash = addressHashProp ?? addressHashParam
   const address = useAppSelector((state) => selectAddressByHash(state, addressHash))
-  const { assets, direction, outputs, lockTime, infoType } = useTransactionInfo(tx, addressHash, showInternalInflows)
+  const { assets, direction, outputs, lockTime, infoType } = getTransactionInfo(tx, showInternalInflows)
```

Comment:
I changed this hook to a simple function so that it can be used in the `applyFilters` method just above (in `TransactionList.tsx`.